src/ark/settings.py

```diff
@@ -4,6 +4,7 @@
 # default cell table column names
 CELL_SIZE = 'cell_size'  # cell size (number of pixels in the cell)
 CELL_LABEL = 'label'  # cell label number (regionprops)
+CELL_SEGMENTATION_LABEL = "segmentation_label"  # cell label number (segmentation)
```

Comment:
Sorry if you're not done making changes yet - but why is there both `CELL_LABEL` and `CELL_SEGMENTATION_LABEL` in the settings file? I assume those are the same thing?

Reply:
@cliu72 

Looks like `"label"` is converted to `"segmentation_label"` in Pixie https://github.com/angelolab/ark-analysis/blob/44b46c44fd9e4e632fc9bdb4bc9a2f491b33df38/src/ark/phenotyping/cell_cluster_utils.py#L107-L109


It's used throughout pixie, here is an example. Not really sure why we use both.
https://github.com/angelolab/ark-analysis/blob/44b46c44fd9e4e632fc9bdb4bc9a2f491b33df38/src/ark/phenotyping/weighted_channel_comp.py#L127-L129
@alex-l-kong could probably give some more insight.

Reply:
Ah I see. I think it'd be better to make it consistent. @alex-l-kong in the comments you say it makes for a simpler merge, but why is it necessary? I don't think there is anything else in the table with the "label" column name

If it's going to be too annoying to change `segmentation_label` to `label`, I think we should remove `CELL_SEGMENTATION_LABEL` from the settings file and just manually specify `label_col = "segmentation_label"` instead (wherever necessary). I think having both `CELL_LABEL` and `CELL_SEGMENTATION_LABEL` in the settings file is confusing.

Reply:
@cliu72 yeah agree that this `'label'` and `'segmentation_label'` discrepancy is annoying. The reason it's done this way is `'label'` is the name of seg label column in the cell table, but `'segmentation_label'` is the name of the equivalent column in the pixel cluster data `.feather` files.

I'd say change the `segmentation_label` to `label` (or vice versa) is best practice, the only annoying thing would be to update the `.merge` logic that happens inside the `c2pc` data function. If you're not comfortable with that @srivarra then I can contribute that change to this PR.

Alternatively, to expedite the review on this PR, we could implement Candace's hotfix, then open a separate issue for standardizing the segmentation label column names.